module/PowerNSX.psm1

```diff
@@ -15548,6 +15639,60 @@ function Remove-NsxEdgeCertificate{
     end {}
 }
 
+#########
+#########
+# Edge DNS functions
+
+function Get-NsxEdgeDNS {
```

Comment:
for another i think ?

i have already write code for this
https://github.com/vmware/powernsx/pull/426 (but no yet merged...)

Reply:
Sure ! no problem ;-) (it is also for avoid to work on the same stuff..)